README.md

```diff
@@ -1 +1,92 @@
-# fcc_tribute_page
+# Robot technology
+
+> The robot technology for who are willing to lean how to build or how to know the year it was discoverd.
+
+![screenshot](./https://github.com/abdulsomod1/fcc_tribute_page/issues/3#issue-1090134207)
```

Comment:
This should be a valid screenshot, see https://github.com/abdulsomod1/fcc_tribute_page/blob/add-tribute-page/README.md